tests/integration/test_vulnerability_detector/test_providers/test_update_interval.py

```diff
@@ -52,12 +51,14 @@
 # Callbacks
 callback_provider_database_updating = make_vuln_callback('Starting .* database update')
 
+
 # Fixtures
 @pytest.fixture(scope='module', params=configurations, ids=ids)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
+
 # Tests
```

Comment:
This one too.

Reply:
Done in 7fcdc7ebecdd546a4957814ec960f6304685d265